starlingx/inventory/v1/cpus/results.go

```diff
@@ -42,15 +42,15 @@ type CPU struct {
 	// Processor is the processor package number.  A host usually has 1 or 2
 	// processors but that number could be higher.  Each is usually associated
 	// to a seperate NUMA socket.
-	Processor int `json:"processor"`
+	Processor int `json:"numa_node"`
```

Comment:
Why are these being changed? Was the mapping wrong?

Reply:
system --debug has field as numa_node